docs/MorganeyIndex.md

```diff
@@ -0,0 +1,32 @@
+# Morganey Index
```

Comment:
I think that you should add a link to this file from some documentation root (probably from the Readme file). Or will you do it later?